roads.mss

```diff
@@ -1419,16 +1417,24 @@ come in as well.
           line-color: @speed20-fill;
       }
       [maxspeed_kmh < 10],
-      [motor_vehicle='no'][bicycle!='no'] {
+      [motor_vehicle='no'][can_bicycle!='no'] {
```

Comment:
I was using `bicycle` here to only check if cycling was allowed (and not general access tags etc). I think this is the only use of `bicycle` in this file, so we can probably remove it from SQL queries.